notes.md

```diff
@@ -0,0 +1,1900 @@
+# PyTorch 2.0 Release notes
+
+# Highlights
+
+- &lt;Summary to be provided>
+
+# Backwards Incompatible changes
+
+### **Set min supported Python version to 3.8 (#93155)**
```

Comment:
nit: this and the next item should have the same format (I prefer Drop Support because it's clearer, but either way).  It's better if it looks like one person wrote this :).